quickfixj-core/src/main/java/quickfix/JdbcStore.java

```diff
@@ -293,23 +299,39 @@ public boolean set(int sequence, String message) throws IOException {
 
     public void setNextSenderMsgSeqNum(int next) throws IOException {
         cache.setNextSenderMsgSeqNum(next);
-        storeSequenceNumbers();
+        storeOutgoingSequenceNumbers();
     }
 
     public void setNextTargetMsgSeqNum(int next) throws IOException {
         cache.setNextTargetMsgSeqNum(next);
-        storeSequenceNumbers();
+        storeIncomingSequenceNumbers();
     }
 
-    private void storeSequenceNumbers() throws IOException {
+    private void storeIncomingSequenceNumbers() throws IOException {
```

Comment:
Just a minor thing but would IMHO be cleaner (with less duplication): keep one method storeSequenceNumbers() but pass in the parameters for the seqnum and the SQL?